docs/concepts/macros/sqlmesh_macros.md

```diff
@@ -260,10 +260,22 @@ It includes three elements:
 2. A value to return if the condition is `TRUE`
 3. A value to return if the condition is `FALSE` [optional]
 
-These elements are specified as `@IF([logical condition], [value if TRUE], [value if FALSE])`.
+These elements are specified as:
+
+```sql linenums="1"
+@IF([logical condition], [value if TRUE], [value if FALSE])
+```
 
 The value to return if the condition is `FALSE` is optional - if it is not provided and the condition is `FALSE`, then the macro has no effect on the resulting query.
 
+It's also possible to use this macro in order to conditionally execute pre/post-statements:
+
+```sql linenums="1"
+@IF([logical condition], [statement to execute if TRUE])
+```
+
+Note that the statement to be executed must not end with a semi-colon.
```

Comment:
Looking at the example below, is it correct that the `@IF()` statement SHOULD end with a semicolon but the statement within should not?

Reply:
That's correct, I can update this note if it doesn't seem clear

Reply:
Updated